pkg/README.md

```diff
@@ -0,0 +1 @@
+The architecture of the directory same as the [go.src](https://github.com/golang/go/tree/ad646b3/src)
```

Comment:
You don't need this file

Reply:
Yes with [v0.29.1 go-astikit.AdaptTestLogger](https://github.com/asticode/go-astikit/blob/4ba43aee86681c99fd72a8ea6758778b554dceaf/logger.go#L183), I don't need this.